src/drom_lib/commandNew.ml

```diff
@@ -16,26 +16,14 @@ open EzFile.OP
 let cmd_name = "new"
 
 let create_project ~config ~name ~skeleton ~mode ~dir ~inplace ~args =
-  let license =
-    match config.config_license with
-    | None -> Skel_licenses.LGPL2.key
-    | Some license -> license
-  in
-  let dir =
-    match dir with
-    | None -> "src" // name
-    | Some dir -> dir
-  in
+  let license = Misc.option_value config.config_license ~default:License.default in
+  let dir = Misc.option_value dir ~default:("src" // name) in
```

Comment:
I am not fond of this kind of refactoring in general: though it makes the code shorter, so a little more readable, it also introduces a possibility of bug and increased computation time, since the `~default` arg is now always executed, when it was only executed in the `None` case previously.